web/src/components/vis/Heatmap.vue

```diff
@@ -220,9 +237,15 @@ export default {
         return r;
       });
 
-      inner.select('text').html(d => (collapsed.has(d.data) ? MDI_PLUS_CIRCLE : MDI_MINUS_CIRCLE));
+      inner.select('text').html((d) => {
+        if (wrapper.focus === d.data) {
+          return MDI_STAR_CIRCLE;
+        }
+        return collapsed.has(d.data) ? MDI_PLUS_CIRCLE : MDI_MINUS_CIRCLE;
+      });
       inner.select('title').text(d => d.data.name);
       inner.classed('collapsed', d => collapsed.has(d.data));
+      inner.classed('focussed', d => wrapper.focus === d.data);
```

Comment:
`focussed` -> `focused`

Both are acceptable, but the latter is much more common in American English.

https://english.stackexchange.com/questions/4791/focussed-or-focused-rules-for-doubling-the-last-consonant-when-adding-ed